packages/tools/examples/volumeSlabScroll/index.ts

```diff
@@ -143,6 +144,21 @@ addButtonToToolbar({
   },
 });
 
+addToggleButtonToToolbar({
+  id: 'slabScroll',
+  title: 'Toggle Slab Scroll',
+  defaultToggle: false,
+  onClick: (toggle) => {
+    let scrollSlabs = false;
```

Comment:
How about instead of using an if do one of the following?

`const scrollSlabs = toggle ? true : false;` 

or

`const scrollSlabs = !!toggle`